src/devtools/client/inspector/markup/new-markup.js

```diff
@@ -97,6 +101,15 @@ class MarkupView {
     return this._eventTooltip;
   }
 
+  /**
+   * An object representing the markup tree. The key to the object represents the object
+   * ID of a NodeFront of a given node. The value of each item in the object contains
+   * an object representing the properties of the given node.
+   */
+  getTree() {
```

Comment:
is this used? I checked because it seems a bit fishy to be accessing the state directly, but in the right circumstances it could be fine

Reply:
I used it initially, but now it isn't used anymore.